### DISCUSSION TITLE
Little Issue (imho)

### DISCUSSION THREAD

**Discussion by ghost** [neon-nyan/Collapse#470] [General]
why is it that when you hover over the "collapse" and "close" buttons, the contour is complete, without rounding, and not rounded like on other buttons?
![image(2)](https://github.com/CollapseLauncher/Collapse/assets/159782653/4f509a0e-0f92-4259-a443-f148c8413192)
![image(1)](https://github.com/CollapseLauncher/Collapse/assets/159782653/8f947711-57b7-4935-aa52-811741c20d5d)
![image](https://github.com/CollapseLauncher/Collapse/assets/159782653/61b1c7ee-8b70-44bd-b90a-58e3bef203cf)

**Comment by Cryotechnic** [comment]
![image](https://github.com/CollapseLauncher/Collapse/assets/9833218/8f06c8d3-aa3f-4dac-8826-bd85acbab5b2)

If you're running Preview, the buttons should look rounded, as seen above. What are you running?

**Reply by ghost** [reply]
A regular shortcut from the desktop, I can make a video

**Reply by ghost** [reply]
And I also want to ask if there is an opportunity to pin game shortcuts for easy movement between games in the launcher, for example here (place in the screenshot)?
![image(3)](https://github.com/CollapseLauncher/Collapse/assets/159782653/c7682500-df64-468e-9499-f96d9496c08b)

**Comment by Cryotechnic** [comment]
> And I also want to ask if there is an opportunity to pin game shortcuts for easy movement between games in the launcher, for example here (place in the screenshot)? ![image(3)](https://private-user-images.githubusercontent.com/159782653/332463527-c7682500-df64-468e-9499-f96d9496c08b.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTYzMDE4ODcsIm5iZiI6MTcxNjMwMTU4NywicGF0aCI6Ii8xNTk3ODI2NTMvMzMyNDYzNTI3LWM3NjgyNTAwLWRmNjQtNDY4ZS05NDk5LWY5NmQ5NDk2YzA4Yi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNTIxJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDUyMVQxNDI2MjdaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1hNDBiNDZmY2M0MWMyMDkzYTQ3M2MyNzRlYmMwNmRiYzA0ZTFlYmZmMGNhNWNmNmFkMDM3MmU5YjA2ZjZjOTgyJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9._jcu4JtArkY2N51sHhoLpFw5QKYMevk3P0x84i6If74)

While I make no promises as to the availability of said feature, we already have an issue tracking it (#438). With ZZZ right around the corner, we're currently focused on bringing support for that title, which takes away most of our time bringing other UI changes like this one.

**Reply by ghost** [reply]
and what about the visual mini-bug that I wrote about before?

**Reply by Cryotechnic** [reply]
What version of Collapse are you currently running?

**Reply by ghost** [reply]
1.73.8

**Comment by Cryotechnic** [comment]
> 1.73.8

I believe that the rounding issue was fixed in Preview. You can try switching to that and see if it resolves the issue.

**Reply by ghost** [reply]
Thank you for such a convenient launcher!
